app/models/task.rb

```diff
@@ -17,10 +17,6 @@ def completed_actions_count
     @completed_actions_count ||= actions.where(action_type: "single-checkbox").where(completed: true).count
   end
 
-  def clear_legal_documents_type?
-    section.title == LEGAL_DOCUMENT_SECTION_TITLE
```

Comment:
There is a `LEGAL_DOCUMENT_SECTION_TITLE` constant at the top of this file that can go in the bin too